zaza/utilities/deployment_env.py

```diff
@@ -130,11 +127,9 @@ def find_setup_file():
     :returns: Location of zaza config file or None if not found.
     :rtype: str or None
     """
-    ctxt = {
-        'home': os.environ.get('HOME')}
     for setup_file in ZAZA_SETUP_FILE_LOCATIONS:
-        if os.path.isfile(setup_file.format(**ctxt)):
-            return setup_file.format(**ctxt)
+        if os.path.isfile(setup_file):
+            return setup_file
```

Comment:
Do the `expandvars()` call here to be consistent with the current semantics of the function; i.e. evaluate the `HOME` env in this function rather than at module load time.